src/ploneintranet/activitystream/browser/activity_provider.py

```diff
@@ -144,10 +136,10 @@ def text(self):
         return text
 
     def is_attachment_supported(self):
-        return IAttachmentStoragable is not None
+        return True
 
     def is_preview_supported(self):
-        return IDocconv is not None
+        return True
 
```

Comment:
Yes!!! That is definetely the goal :)
I was setting up a task to address this (see wb 4460).
I wanted this pull request to be very specific in removing the ImportError checks.